whylog/config/__init__.py

```diff
@@ -117,13 +118,16 @@ def create_file_with_config_paths(cls, config_paths, path_to_config):
 
 @six.add_metaclass(ABCMeta)
 class AbstractConfig(object):
+    WORDS_COUNT_IN_NAME = 4
```

Comment:
This is a default value that the user should be able to override in configuration, not the final setting.